### DISCUSSION TITLE
How to export ipynb to html with R kernel?

### DISCUSSION THREAD

**Discussion by amssljc** [microsoft/vscode-jupyter#9565] [Questions and Answers] [answered]
I can export my ipynb as .py .pdf. .html document with python kernel. But when I try to use R kernel, there is no this option "export"? I want to know what I can do? Or is it a bug??

**Comment by DonJayamanne** [comment, marked as answer]
> Or is it a bug??

Its a bug, closing in favor of https://github.com/microsoft/vscode-jupyter/issues/9571
